packages/diagrams/frontend/sirius-components-diagrams/src/renderer/direct-edit/DiagramDirectEditContext.types.ts

```diff
@@ -13,24 +13,24 @@
 
 export interface DiagramDirectEditContextValue {
   currentlyEditedLabelId: string | null;
-  editingKey: string | null;
+  editingInput: string | null;
   directEditTrigger: DirectEditTrigger | null;
   setCurrentlyEditedLabelId: (
     directEditTrigger: DirectEditTrigger,
     currentlyEditedLabelId: string,
-    editingKey: string | null
+    editingInput: string | null
   ) => void;
   resetDirectEdit: () => void;
 }
 
-export type DirectEditTrigger = 'keyDown' | 'F2' | 'palette' | 'doubleClick';
+export type DirectEditTrigger = 'keyDown' | 'F2' | 'palette' | 'doubleClick' | 'paste';
```

Comment:
It is unclear to me where this value is used, I added a new one anyways to be consistent with the existing code.